src/test/java/com/google/api/codegen/testdata/discoveries/bigquery.v2.json.overrides

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I intend to come back to this in a different way soon. I need to look more closely at what's going on at the `DiscoveryImporter` level that sets this resource apart from others.
It will not be forgotten!